kitsune/settings.py

```diff
@@ -757,6 +757,8 @@ def JINJA_CONFIG():
     "SESSION_SERIALIZER", default="django.contrib.sessions.serializers.PickleSerializer"
 )
 
+# CSRF
+CSRF_COOKIE_SECURE = config("CSRF_COOKIE_SECURE", default=True, cast=bool)
```

Comment:
Perhaps this should be `default=not DEBUG` like `SESSION_COOKIE_SECURE` above?

Or maybe both should be `default=True` so they fail-safe? Firefox seems to send Secure cookies to localhost anyway.